src/IO/AssimpLoader/AssimpCameraDataLoader.cpp

```diff
@@ -97,9 +97,8 @@ void AssimpCameraDataLoader::loadCameraData( const aiScene* scene,
 }
 
 Core::Matrix4 AssimpCameraDataLoader::loadCameraFrame( const aiScene* scene,
-                                                       const Core::Matrix4& parentFrame,
-                                                       const aiCamera& cameraNode,
-                                                       Camera& data ) const {
+                                                       const Core::Matrix4& /*parentFrame*/,
+                                                       const aiCamera& cameraNode ) const {
     // old version
```

Comment:
If this code is no more needed, it should be deleted.

Reply:
actually, keep the old version for the record, if someone as a issue with camera loading, it's here to investigate easilly.